assets/src/edit-story/components/panels/textStyle/font.js

```diff
@@ -83,11 +86,22 @@ function FontControls({ selectedElements, pushUpdate }) {
                     ? b
                     : a
                 );
+
+              const fontObj = fonts.find((item) => item.value === value);
+
               pushUpdate(
                 {
-                  fontFamily: value,
+                  font: {
+                    family: value,
+                    ...objectPick(fontObj, [
+                      'service',
+                      'fallbacks',
+                      'weights',
```

Comment:
Ok. I'm slightly confused: why do we need both `weights` and `getFontWeights`?

Reply:
Yeah I think `getFontWeights` is kinda obsolete now. I‘ll take a look.

Reply:
Removed it from the callback here